spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java

```diff
@@ -365,6 +369,29 @@ public boolean updateInstanceStatus(String serviceId, String instanceId, String
     }
   }
 
+  public MicroserviceResponse getServices() throws ServiceCombException {
+    MicroserviceResponse result = null;
+    Response response = null;
+    try {
+      response = httpTransport.sendGetRequest(
+          buildURI("/registry/microservices"));
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        LOGGER.info(response.getContent());
```

Comment:
response.getContent() may  be needed more time costly, and can add if condition like this :if(isdebug)(debug(info))

Reply:
done